codes/quantum/qubits/two_dimensional_hyperbolic_surface.yml

```diff
@@ -0,0 +1,106 @@
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: two_dimensional_hyperbolic_surface
+physical: bits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Two-dimensional hyperbolic surface code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:1506.04029}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Hyperbolic surface codes are closely related to the toric code (aka surface code) except the tessellations on the manifold are derived on a hyperbolic geometry (ie, non-Euclidean geometry, think saddle surfaces when defined on a 2D plane) instead of Eulicdean surfaces. For a tessellation involving regular polygons with \( r \) sides and \( s \) polygons meeting at each edge, the number of logical qubits (which sit at the center of each tile) is given by \( k = (1-2/r - 2/s) n + 2 \). Some possible tilings are {r,s}: {7,3}, {5,4}. For closed surfaces, two-dimensional hyperbolic surface codes have an asymptotically constant encoding rate $k/n$ with a distance scaling logarithmically $\log(n)$. Additionally, surfaces with more tiling (larger values of r, s) have higher encoding rates (more logical qubits per physical qubits) but stabilizers with larger weights. Given the stabilizers are measured on noisy circuits, lower weight stabilizers are preferable. Therefore, the family of hyperbolic codes generated from the {5,4} tiling may be more favorable since they have lower weight stabilizers. '
+
+protection: 'Protects against Pauli errors with distance \( d \propto \log(n) \). Approximate code parameters are \( [[n, (1-2/r - 2/s)  n + 2, \log(n) ]] \)'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: '\( k/n = (1-2/r - 2/s) + 2/n \) which approaches a constant value as the number of physical qubits grows. The distance of the code scales logarithmically with \( n \) and the weight of the stabilizers is \( r \) for \(r\)-checks and \(s\) for \(X\)-checks, ( Recall \( {r,s} \) are the tiling parameters for the hjyperbolic surface)'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders:
```

Comment:
I'm not sure either of these deal with encodings. Discussion here should be about any unitary, dissipative, or measurement-based preps that are special/unique to this family. (E.g., stabilizer-based encodings are already covered in stabilizer code entry.) If you can't find any after a bit of searching, then comment this section out.

Reply:
* Removed these two points and made sure the ideas were included in other sections. 
* Couldn't find anything about encodings that are specific to the two-dimensional hyperbolic surface code. 
* Commented this section out